config/core/test-configs.yaml

```diff
@@ -84,7 +84,7 @@ file_systems:
 
   debian_buster-ltp_nfs:
     type: debian
-    ramdisk: 'buster-ltp/20211203.0/{arch}/initrd.cpio.gz'
+    ramdisk: 'buster-ltp/20211203.0/{arch}/rootfs.cpio.gz'
```

Comment:
The `initrd.cpio.gz` is created especially to then be able to switch the root file system to NFS.  So this change can't be done, the `rootfs.cpio.gz` is to boot with just a ramdisk which has a full root file system.

We either need to fix NFS root on QEMU or make a variant for LTP tests to run on QEMU with a ramdisk.

Reply:
I see. I will try to add a new variant for this case.

Reply:
Hi @gctucker 
I modified a new version according to your suggestion.
Please help to check it.
Thanks.